templates.go

```diff
@@ -12,7 +12,10 @@ func loadTemplates(dir string) *template.Template {
 		return getTemplates()
 	}
 	logger.Printf("using custom template directory %q", dir)
+        //<!--COCO Begin-->
+        //COCO: Added new template logout.html to parse it
```

Comment:
- comments also needs to be aligned with accordingly

Reply:
handled